### PR TITLE
[SR] Translation of README

### PR DESCRIPTION
**Pull request by EmaPajic**

### CODE REVIEW

File: docs/rs/README-RS.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
+# Duboko učenje (u PyTorch-u) [![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/Atcold/pytorch-Deep-Learning/master)
+
+Ovaj repozitorijum ima [prateći sajt](https://atcold.github.io/pytorch-Deep-Learning/), gde se svi materijali kursa mogu naći u video i tekstualnom formatu.
```
Comment (jecaJeca) [inline docs/rs/README-RS.md:3]: Mislim da ne treba zarez. Može da se kaže i - gde se svi materijali kursa mogu naći u originalnom video i tekstualnom formatu
Čisto da bude jasno da je engleska verzija a ne srpska.

File: docs/rs/README-RS.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+
+# Podešavanja
+
+Da bi bilo moguće pratiti vežbe, potreban je laptop sa instaliranom Miniconda-om (minimalnom verzijom Anaconda-e) i nekoliko paketa jezika Python.
+Naredne instrukcije rade za Mac i Ubuntu Linux, a za Windows je potrebno da se instalira i radi u [Git BASH](https://gitforwindows.org/) terminalu.
+
+
+## Instalacija
```
Comment (jecaJeca) [inline docs/rs/README-RS.md:15]: Može i _Podešavanje radnog okruženja_

File: docs/rs/README-RS.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+```bash
+# cd pytorch-Deep-Learning
+conda env create -f environment.yml
+source activate pDL
+```
+
+
+## Startovati Jupyter Notebook ili JupyterLab
+
+Startovati iz terminala kao uobičajeno:
```
Comment (jecaJeca) [inline docs/rs/README-RS.md:48]: Možda bez ovoga?

File: docs/rs/README-RS.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+Ili, za klasični interfejs:
+
+```bash
+jupyter notebook
+```
+
+
+## Vizuelizacija Notebook-a
+
+*Jupyter Notebooks* se koriste kroz lekcije za interaktivnu eksploraciju i vizualizaciju.
```
Comment (jecaJeca) [inline docs/rs/README-RS.md:63]: interaktivno ispitivanje (analizu) podataka i vizuelizaciju

File: docs/rs/README-RS.md [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+```
+
+
+## Vizuelizacija Notebook-a
+
+*Jupyter Notebooks* se koriste kroz lekcije za interaktivnu eksploraciju i vizualizaciju. 
+
+Koristimo tamni stil i za *GitHub* i *Jupyter Notebook*.
+Trebalo bi podesiti isto jer inače neće izgledati lepo.
+JupyterLab ima ugrađenu tamnu temu koju je moguće odabrati, pa je potrebno instalirati nešto samo ukoliko želite da koristite klasičan interfejs notebook-a.
```
Comment (jecaJeca) [inline docs/rs/README-RS.md:67]: pa je potrebno izvršiti dodatne instalacija samo ukoliko ...

File: docs/rs/index.md [hunk 6]
```diff
@@ -0,0 +1,343 @@
...
+| PREDAVAČI   | Yann LeCun & Alfredo Canziani |
+| LEKCIJE     | Ponedeljak 16:55 – 18:35, [GCASL C95](http://library.nyu.edu/services/campus-media/classrooms/gcasl-c95/) |
+| VEŽBE       | Utorak 19:10 – 20:00, [GCASL C95](http://library.nyu.edu/services/campus-media/classrooms/gcasl-c95/) |
+| [PIAZZA](https://piazza.com/nyu/spring2020/dsga1008/home)      | Pristupni kod: `DLSP20` |
+| MATERIJALI  | [Google Drive](https://bitly.com/DLSP20), [Notebooks](https://github.com/Atcold/pytorch-Deep-Learning) |
+
+
+## Opis
+
+Ovaj kurs prati najskorije tehnike u dubokom učenju i učenju reprezentacije. Fokusira se na nadgledano i nenadgledano učenje, embedding metode, učenje metrika, konvolucione i rekurentne neuronske mreže, sa primenama u kompjuterskoj viziji, razumevanju prirodnog jezika i prepoznavanju govora. Potrebno predznanje: [DS-GA 1001 Intro to Data Science](https://cds.nyu.edu/academics/ms-curriculum/) ili kurs mašinskog učenja na nivou master ili doktorskih studija.
```
Comment (jecaJeca) [inline docs/rs/index.md:19]: je li ovo ispravan prevod graduate-level-a

File: docs/rs/README-RS.md (hunk 1 above)
Comment (EmaPajic) [reply docs/rs/README-RS.md:3]: Tehnički, koliko ja shvatam, na sajtu će moći da se vidi i verzija na srpskom. Ako pogledaš sajt, može da se promeni jezik.

File: docs/rs/README-RS.md (hunk 2 above)
Comment (EmaPajic) [reply docs/rs/README-RS.md:15]: resolved

File: docs/rs/README-RS.md (hunk 3 above)
Comment (EmaPajic) [reply docs/rs/README-RS.md:48]: resolved

File: docs/rs/README-RS.md (hunk 4 above)
Comment (EmaPajic) [reply docs/rs/README-RS.md:63]: resolved

File: docs/rs/README-RS.md (hunk 5 above)
Comment (EmaPajic) [reply docs/rs/README-RS.md:67]: resolved

File: docs/rs/index.md (hunk 6 above)
Comment (EmaPajic) [reply docs/rs/index.md:19]: Nisam sigurna kako bi se najpravilnije prevelo, ali van Srbije obično se vodi undergraduate i graduate level pa pretpostavljam da je ok da se kaže ovako

**Review by Atcold — CHANGES_REQUESTED** [review]
Very good! Let's use `sr` for Serbian (see [our website template](https://github.com/ebetica/jekyllbook/blob/bdfa5ba3ac15eb9e9f74cc17178cc5b686c6c6ba/static/book.js#L313)).
Moreover, you need to update `docs/_config.yml`.
Kindly rename `README-RS.md` into `README-SR.md`.

**Comment by EmaPajic** [comment]
> Very good! Let's use `sr` for Serbian (see [our website template](https://github.com/ebetica/jekyllbook/blob/bdfa5ba3ac15eb9e9f74cc17178cc5b686c6c6ba/static/book.js#L313)).
> Moreover, you need to update `docs/_config.yml`.
> Kindly rename `README-RS.md` into `README-SR.md`.

Hi, sure, I'll change it. Actually, I wrote sr initially but noticed that that flag for sr is wrong and that Serbian flag is created with rs.

**Comment by Atcold** [comment]
Hi @EmaPajic. I'm a little confused about your "I'll change it".
`docs/_config.yml` and all the other `README.md`s have not yet been updated.
Am I missing something?

**Comment by EmaPajic** [comment]
> Hi @EmaPajic. I'm a little confused about your "I'll change it".
> `docs/_config.yml` and all the other `README.md`s have not yet been updated.
> Am I missing something?

Updated now

**Review by Atcold — APPROVED** [review]
Thanks, merging. I'll update the `README.md` for all other languages myself…

**Pull request merged**